### ISSUE TITLE
[Development Call]: GitHub copilot in RStudio

### ISSUE THREAD

**Issue by alexandervossen** [tilburgsciencehub/website#937]
### Is your content request related to a problem you've encountered during your research process? Please describe.

I found out yesterday that RStudio now spports GitHub copilot. https://docs.posit.co/ide/user/ide/guide/tools/copilot.html 
I am looking for someone who writes a short setup & tutorial with me for it.  Also I am still looking for someone to help with #866    so feel free to reach out. 

### Would you like this to be a Building Block or a Tutorial?

Tutorial

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by VirginiaMirabile** [comment]
Hi Alexander, happy to join this project!

**Comment by VirginiaMirabile** [comment]
Hi Alex, 
I wanted to give you an update on the content for the tutorial. 

-  I have outlined the different sections that are going to constitute the tutorial
- Wrote the introduction section 
- Outlined what is GitHub Copilot, it's features and possible uses in R 
- Developed a section explaining how to set up RStudio, GitHub and how to request access to the tool for free (I have also used links of building blocks on these topics from the TSH website) 

I am waiting for the access to Copilot, in the meantime I'm watching some youtube videos on the topic!

Once I get the approval, the next section will be focused on a few examples on how to use Copilot in R with ggplot2.
